course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/CourseBatchUtil.java

```diff
@@ -63,7 +63,7 @@ public static void syncCourseBatchForeground(RequestContext requestContext, Stri
             + esResponse);
   }
 
-  public static Map<String, Object> validateCourseBatch(RequestContext requestContext, String courseId, String batchId) {
+  public static Map<String, Object> validateCourseBatch(RequestContext requestContext, String courseId, String batchId, String eventId) {
```

Comment:
this method is not called from your newly added methods for event cert handling... so do revert this...